discord/scheduled_event.py

```diff
@@ -131,7 +131,7 @@ def _update(self, data: GuildScheduledEventPayload) -> None:
         self.id: int = int(data['id'])
         self.guild_id: int = int(data['guild_id'])
         self.name: str = data['name']
-        self.description: str = data.get('description', '')
+        self.description: Optional[str] = data.get('description', '')
```

Comment:
This still has the default of `''`.

Reply:
alright, wasn't sure if I should keep that or not